daffodil-cli/src/windows/apache-daffodil.iss

```diff
@@ -0,0 +1,102 @@
+; Licensed to the Apache Software Foundation (ASF) under one or more
+; contributor license agreements.  See the NOTICE file distributed with
+; this work for additional information regarding copyright ownership.
+; The ASF licenses this file to You under the Apache License, Version 2.0
+; (the "License"); you may not use this file except in compliance with
+; the License.  You may obtain a copy of the License at
+;
+;     http://www.apache.org/licenses/LICENSE-2.0
+;
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS,
+; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+; See the License for the specific language governing permissions and
+; limitations under the License.
+
+[Setup]
+; Do not change the AppID, this GUID uniquely identifies Daffodil
+AppId={{4C966AFF-585E-4E17-8CC2-059FD70FEC77}
```

Comment:
Is there not an extra '{' at the start here?  Or am I missing something?

Reply:
In Inno Setup braces are used for variables constants. For example, `AppVersion={#VERSION}` below says to set the property to the "VERSION" preprocessor variable that we pass in with the `/D` option in build.sbt.

The double curly braces escapes that and says this isn't actually a variable but is a literal curly brace. Our AppID is a GUID, I think the curly braces need to be wrapped around that for Inno Setup to know it's a GUID? I'm not really sure what the curly braces are for in this case, but I think they are required.

Reply:
I guess you don't need to escape the closing  '}'.  The unbalanced braces just jumped out at me.

Reply:
Yeah, I think you can escape it, but I think it's not necessary so it doesn't complain. I originally went through the inno setup wizard to create the .iss file and this is how it generated AppId it so I just stuck with it